src/component/user/Login/Login.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect } from 'react';
+import classnames from 'classnames';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { useTheme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+
+import AuthenticationContainer from '../authentication-container';
+import ConditionallyRender from '../../common/ConditionallyRender';
+
+import { ReactComponent as SwitchesSVG } from '../../../icons/switches.svg';
```

Comment:
Could we use the Unleash Logo instead, possible an inverted version to make it work with the dark background?

Reply:
Yep. I'll add that.